src/Core/Messaging.php

```diff
@@ -57,10 +57,8 @@ public function setMessages( array $ffMessages ) {
 			}
 			$message = $mail->parseWikiText( ContentCore::parseTitle( trim( $exploded[2] ), true ) );
 			$title = $mail->parseWikiText( ContentCore::parseTitle( trim( $exploded[3] ), true ) );
+			$title = str_replace( [ '<p>', '</p>' ], '', $title );
 			$persistent = ContentCore::parseTitle( trim( $exploded[4] ), true );
-			if ( !in_array( strtolower( $persistent ), self::$messagePersistance ) ) {
```

Comment:
If you remove this here, I believe the messagePersistance array is not needed at all anymore?